task.go

```diff
@@ -165,8 +165,16 @@ func (e *Executor) runDeps(ctx context.Context, call Call) error {
 			if err != nil {
 				return err
 			}
+			depVars := make(Vars, len(d.Vars))
+			for k, v := range d.Vars {
+				v, err := e.ReplaceVariables(v, call)
```

Comment:
As an FYI, it would be good with some restructuring to avoid code duplication in command and deps handling.

This commit only makes things work.